api/SVGDiscardElement.json

```diff
@@ -31,7 +31,7 @@
         "status": {
           "experimental": false,
           "standard_track": true,
-          "deprecated": true
+          "deprecated": false
```

Comment:
@Elchi3 The element was removed from Chrome in 81 and never added to other browser, which is why it is marked as deprecated. I can't find evidence that it should still be deprecated.

Should it be become "experimental" again? It is a bit like a new API after this much time. I guess that would make it confusing though.

Reply:
> I can't find evidence that it should still be deprecated.

Good point, I don't find corresponding wording in the spec that supports the deprecated status.

> Should it be become "experimental" again?

Yes, please set `experimental: true`. (I think `deprecated: true` implies `experimental: true`.)